src/extensions/default/bramble/main.js

```diff
@@ -155,6 +157,8 @@ define(function (require, exports, module) {
                     // Setup the iframe browser and Blob URL live dev servers and
                     // load the initial document into the preview.
                     startLiveDev();
+					
+					BrambleCodeSnippets.init();
```

Comment:
Some indenting issues with this line.  Make sure it lines up with what's on line 159 above.